File: crates/deslop/tests/fixtures/fused-golden-rust/rename_b.rs
pub fn dispatch(mass: i64, span: i64, handler: &str) -> String {
    let rating = mass * 3 + span;
    if rating > 900 {
        return handler.to_owned() + "-freight";
    }
    if rating > 400 {
        return handler.to_owned() + "-ground";
    }
    handler.to_owned() + "-parcel"
}

File: crates/deslop/tests/fixtures/fused-golden-typescript/rename_b.ts
export function dispatch(mass: number, span: number, handler: string): string {
  const rating = mass * 3 + span;
  if (rating > 900) {
    return handler + "-freight";
  }
  if (rating > 400) {
    return handler + "-ground";
  }
  return handler + "-parcel";
}
